plugins/flespiConnector.ts

```diff
@@ -183,7 +253,8 @@ function generateRandomId (length: number): string {
 const plugin: Plugin = (_context: Context, inject: Inject) => {
     inject("initiateClient", createClient);
     inject("getPositionData", setupClient);
-    inject("getDeviceList", getDevices);
+    inject("getChannelList", getAllChannels);
+    inject("getAllDevices", getAllDevices);
```

Comment:
```suggestion
    inject("getDeviceList", getAllDevices);
```
for consistency with channels.